import java.sql.SQLException;

public class JdbcTemplate {

    public static void mainTEst(String[] args) throws SQLException, DatabaseLayerException {

        //private UserRepository userRepo;

        //Statement statement = null;
        //try {
            //statement = JDBCConnection.getInstance().getStatement();
        //} catch (DatabaseLayerException e) {
           // e.printStackTrace();
        //}
        //SpringApplication.run(JdbcTemplate.class, args);

        //UserDTO userDTO = getUserWithJPA("test", "test");
        //System.out.println(userDTO);
        //try{
            //statement.executeQuery("INSERT INTO " + "'coll'.'user'" + "(e_mail, password, street, house_number, plz, city, phone, role, firstname,lastname)" + "VALUES ('abc', 'abc', 'abc', 'abc', 'abc', 'abc', 'abc', 'abc', 'abc')");

        /*
        ResultSet set = null;
        set = statement.executeQuery(
                "SELECT * "
                        + "FROM coll.user "
                        );


        //}
        //catch (SQLException ex) {
           // DatabaseLayerException e = new DatabaseLayerException("Fehler im SQL-Befehl!");
            //System.out.println("Fehler in der SQL Query");
        //}

         */

    }
}
